jwst/datamodels/model_base.py

```diff
@@ -74,6 +74,11 @@ def __init__(self, init=None, schema=None,
         strict_validation: if true, an schema validation errors will generate
             an excption. If false, they will generate a warning.
```

Comment:
Not a functional problem, but while you're modifying this module anyway, we should fix the typos in lines 74-75. Change to: "if true, schema validation errors will generate an exception. If false, ..."

Reply:
Will do. Actually I meant to fix all docstrings in the `__init__` since they are not correctly formatted.